src/schema/city/index.ts

```diff
@@ -42,19 +42,29 @@ const CityType = new GraphQLObjectType<any, ResolverContext>({
       type: showConnection,
       args: pageable({
         sort: PartnerShowSorts,
-        status: EventStatus,
+        status: {
+          type: EventStatus.type,
+          defaultValue: "CURRENT",
+          description: "By default we filter shows by current",
+        },
         discoverable: {
           type: GraphQLBoolean,
           description:
             "Whether to include local discovery stubs as well as displayable shows",
         },
       }),
       resolve: async (city, args, { showsWithHeadersLoader }) => {
+        // default Enum value for status is not properly resolved
+        // so we have to manually resolve it by lowercasing the value
+        // https://github.com/apollographql/graphql-tools/issues/715
+        args.status = args.status.toLowerCase()
```

Comment:
Bummer! Would it be simpler to just set the default value to `"current"` since that is [the actual value](https://github.com/artsy/metaphysics/blob/master/src/schema/input_fields/event_status.ts#L28)? That would make this line unnecessary.

I'm not sure what the best practice is here 🤷‍♂️

Reply:
Strictly speaking that would be wrong at GraphQL level (because the enum is ALL_CAPS) so you'd end up with a schema that looks like:

```
# By default we filter shows by current
status: EventStatus = current
```

Which isn't actually something that EventStatus could be, even though the value under the hood is actually the lowercase string